internal/pkg/scaffold/olm-catalog/csv_updaters.go

```diff
@@ -240,40 +241,82 @@ func (u *InstallStrategyUpdate) updateDeploymentSpecs(strat *olminstall.Strategy
 
 type CustomResourceDefinitionsUpdate struct {
 	*olmapiv1alpha1.CustomResourceDefinitions
-	crKinds map[string]struct{}
+	crIDs map[string]struct{}
 }
 
 func (store *updaterStore) AddOwnedCRD(yamlDoc []byte) error {
 	crd := &apiextv1beta1.CustomResourceDefinition{}
 	if err := yaml.Unmarshal(yamlDoc, crd); err != nil {
 		return err
 	}
-	store.crds.Owned = append(store.crds.Owned, olmapiv1alpha1.CRDDescription{
-		Name:    crd.ObjectMeta.Name,
-		Version: crd.Spec.Version,
-		Kind:    crd.Spec.Names.Kind,
-	})
-	store.crds.crKinds[crd.Spec.Names.Kind] = struct{}{}
+	versions, err := getCRDVersions(crd)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get owned CRD %s versions", crd.GetName())
+	}
+	for _, ver := range versions {
+		kind := crd.Spec.Names.Kind
+		crdDesc := olmapiv1alpha1.CRDDescription{
+			Name:    crd.ObjectMeta.Name,
+			Version: ver,
+			Kind:    kind,
```

Comment:
Nit: Move Line 257 to 261
https://github.com/operator-framework/operator-sdk/blob/089fe3c0154732d5de0ef4fc5db57d88100d292e/internal/pkg/scaffold/olm-catalog/csv_updaters.go#L257-L261

```suggestion
			Kind:    crd.Spec.Names.Kind,
```

Reply:
`kind` will be used below in #1162 so I figured why not add it here.